app/src/androidTest/java/com/lucas/medicaltools/service/MedicalToolsRepositoryTest.kt

```diff
@@ -0,0 +1,96 @@
+package com.lucas.medicaltools.service
+
+import retrofit2.Response
+
+import com.lucas.medical_equip.repository.MedicalTool
+import com.lucas.medical_equip.service.APIService
+import com.lucas.medical_equip.service.RetrofitBuilder
+import com.lucas.medicaltools.MainViewState
+import com.lucas.medicaltools.repository.MedicalToolsRepository
+import io.reactivex.Observable
+import io.reactivex.android.plugins.RxAndroidPlugins
+import io.reactivex.plugins.RxJavaPlugins
+import io.reactivex.schedulers.Schedulers
+import org.junit.After
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import org.mockito.ArgumentMatchers
+
+import org.mockito.Mock
+import org.mockito.Mockito
+import org.mockito.MockitoAnnotations
+import org.mockito.junit.MockitoJUnitRunner
+
+@RunWith(JUnit4::class)
+class MedicalToolsRepositoryTest  {
```

Comment:
Remember what you testing here. You are trying to test that you repo is working as expected....

So
1. Do not use that actual service, your repo needs to use mocks internally to serve files to test different states of your repo.
2. Change your repo to non static please
3. Your repo has dependencies on the Retrofit builder? So what do you do, you need to mock that builder so that it can return different things such as an empty list for example or a list with many entries or errors.
4. Think about testing? Like you want to ensure you have... Code coverage, you have covered your error states, your success states.
5. When you do this.... you enable yourself to truly ensure all the code you've written is correct.